fs/expose/dokan/__init__.py

```diff
@@ -262,12 +309,13 @@ def wrapper(self,*args):
 
 MIN_FH = 100
 
+
 class FSOperations(object):
     """Object delegating all DOKAN_OPERATIONS pointers to an FS object."""
 
-    def __init__(self, fs, fsname="Dokan FS", volname="Dokan Volume"):
+    def __init__(self, fs, fsname="NTFS", volname="Dokan Volume"):
```

Comment:
Ah, interesting. So given that pyfilesystem is basically a "lowest common denominator" FS abstraction, would it be more sensible to use "FAT" here instead of "NTFS"? Or would that "turn off" features that we're actually using?